scripts/build_ses.sh

```diff
@@ -0,0 +1,241 @@
+#!/bin/bash
+# for ses5 install instructions on which this was based see: http://beta.suse.com/private/SUSE-Storage-Beta/doc/ses-manual_en/ceph.install.saltstack.html
+NUM_MASTER=1
+NUM_ADMIN=1
+NUM_MON=3
+NUM_MDS=1
+NUM_IGW=0
+NUM_RGW=2
+
+if [ $# -lt 2 ]
+then
+  echo "Usage: $0 <salt master> <salt minion> [, <salt minion>, ..]"
+  exit 1
+fi
+
+master=$1
+shift
+minions=($@)
+
+function run_ssh_cmd() {
+  host=$1
+  shift
+  ssh $host -C "$@"
+  if [ $? -gt 0 ]
+  then
+    echo "Ssh command '$@' failed on host '$host'"
+    exit 2
+  fi
```

Comment:
bashate is wants indention to be a multiple of 4 (see travis result and/or run `make bashate` locally)